components/api/src/main/java/com/hotels/styx/api/messages/FullHttpResponse.java

```diff
@@ -134,12 +133,12 @@ public Builder newBuilder() {
         return new Builder(this);
     }
 
-    public HttpResponseStatus status() {
+    public int status() {
```

Comment:
This is a good step forward to remove Netty & Google dependencies. However I would still prefer to represent HTTP status code as a separate class rather than just `int`. This is because a class can hold both the status code number and the textual description in one unit, which will pay dividends for example in logging:

say:

` println("status=%s", response.status())`

Which would now show just `status=200` instead of `status=200 OK`.